academic-services/graph/reference-data-schema.md

```diff
@@ -85,6 +89,17 @@ Column # | Name | Type | Note
 6 | CitationCount | long |
 7 | CreatedDate | DateTime |
 
+## EntityRelatedEntities.txt
+
+Column # | Name | Type | Note
+--- | --- | --- | ---
+1 | EntityId | long |
+2 | EntityType | string | af: Affiliation <br> j: Journal <br> c: Conference
+3 | RelatedEntityId | long |
+4 | RelatedEntityType | string | af: Affiliation <br> j: Journal <br> c: Conference
+5 | RelatedType | int | 0: Two entities are similar if they appear on the same paper <br> 1: Two entities are similar if they have common coauthors <br> 2: Two entities are similar if they are co-cited by others <br> 3: Two entities are similar if they have common fields of study <br> 4: Two entities are similar if they appear in the same venue <br> 5: Entity A is similar to entity B if A cites B <br> 6: Entity A is similar to entity B if A is cited by B
```

Comment:
Shouldn't "RelatedType" be "RelationshipType"?

Reply:
I agree. But it is RelatedType in the script. I will keep it as is.